roles/metering/tasks/container-Debian.yml

```diff
@@ -0,0 +1,37 @@
+---
+- name: Create required directories
+  become: true
+  ansible.builtin.file:
+    path: "{{ item }}"
+    state: directory
+    owner: "{{ operator_user }}"
+    group: "{{ operator_group }}"
+    mode: 0750
+  loop:
+    - "{{ metering_configuration_directory }}"
+    - "{{ metering_data_directory }}"
+    - "{{ metering_docker_compose_directory }}"
+
+- name: Copy settngs.conf file
+  ansible.builtin.template:
+    src: settings.conf.j2
+    dest: "{{ metering_configuration_directory }}/settings.conf"
+    owner: "{{ operator_user }}"
+    group: "{{ operator_group }}"
+    mode: 0640
```

Comment:
Is a restart of the service required when changing this file? If yes add a restart handler here.